_posts/2022-03-07-using-private-image-repositories.markdown

```diff
@@ -0,0 +1,201 @@
+---
+layout: post
+title:  "Deploying workloads with images from private repositories"
+date:   2022-02-07 17:00:00 +0100
+categories: flotta
+author: Jakub Dżon
+tags:
+  - guide
+  - flotta
+  
+summary-1: Deploying workloads with images from private repositories
+---
+
+Users can deploy their containerized workloads to edge devices using Flotta and very often images they want to use are 
+not public and are available only for users or services having proper credentials. 
+
+Flotta supports that use case and allows users to use private images in their `EdgeDeployments`; this post shows how to 
+make use of that support by walking you through all the required steps.
+
+### Prerequisites
+
+1. `podman` installed
```

Comment:
Do you think it makes sense to link to other documentation which explains how to meet prereqs?

Reply:
Good idea.